src/api/app/models/token/workflow.rb

```diff
@@ -20,9 +19,9 @@ def call(options)
     return validation_errors unless validation_errors.none?
 
     # This is just an initial generic report to give a feedback asap. Initial status pending
-    ScmInitialStatusReporter.new(@scm_webhook.payload, @scm_webhook.payload, scm_token, workflow_run).call
+    ScmInitialStatusReporter.new(@scm_webhook.payload, @scm_webhook.payload, scm_token, workflow_run.reload).call
```

Comment:
Why reload?

Reply:
used it at some point, no reason anymore, I removed it.